src/EA36E0.c

```diff
@@ -1,18 +1,57 @@
 #include "dead.h"
 #include "common.h"
 
-INCLUDE_ASM(s32, "EA36E0", func_80242DE0_EA36E0);
+#define NAMESPACE EA36E0
+
+// Copy of kzn_17?
+
+#define UNK_FUNC_50_VAR1 -583.0
+#define UNK_FUNC_50_VAR2 165.0
```

Comment:
Why are these macros?

Reply:
Ah, I see - consider deduplicating these? This map might be a dead version of kzn_17, then.

Reply:
The function is used in 2 exisiting maps, but with different doubles, and then there is also the dead version with the same doubles, so I made the file a .inc.c with these macros instead.

Reply:
Ah I missed you already deduplicated them my bad